src/App.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { FC } from "react"
+import { ErrorBoundary } from 'react-error-boundary'
```

Comment:
`ErrorBoundary` should be implemented manually. According to the task: `Implement and use <ErrorBoundary /> component for catching and displaying errors.`

Please use [this](https://reactjs.org/docs/error-boundaries.html).